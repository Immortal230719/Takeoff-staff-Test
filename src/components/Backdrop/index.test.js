import React from 'react';
import { shallow } from 'enzyme';
// import Adapter from 'enzyme-adapter-react-16';
import { CustomBackdrop } from './index';

describe('Backdrop', () => {
  it('should render correctly', () => {
    const component = shallow(<CustomBackdrop show />);

    expect(component).toMatchSnapshot();
  });

  it("shouldn't render", () => {
    const component = shallow(<CustomBackdrop show={false} />);

    expect(component).toMatchSnapshot();
  });
});
