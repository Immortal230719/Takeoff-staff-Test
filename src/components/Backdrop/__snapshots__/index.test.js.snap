// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Backdrop should render correctly 1`] = `
<WithStyles(ForwardRef(Backdrop))
  className="makeStyles-backdrop-1"
  open={true}
>
  <WithStyles(ForwardRef(CircularProgress))
    color="inherit"
  />
</WithStyles(ForwardRef(Backdrop))>
`;

exports[`Backdrop shouldn't render 1`] = `
<WithStyles(ForwardRef(Backdrop))
  className="makeStyles-backdrop-1"
  open={false}
>
  <WithStyles(ForwardRef(CircularProgress))
    color="inherit"
  />
</WithStyles(ForwardRef(Backdrop))>
`;
